src/con_duct/suite/ls.py

```diff
@@ -98,6 +99,15 @@ def load_duct_runs(
     return loaded
 
 
+def ensure_compliant_schema(info_dict):
+    if parse_version(info_dict["schema_version"]) == parse_version(__schema_version__):
+        return
+    # working_directory added in 0.2.1
+    elif parse_version(info_dict["schema_version"]) < parse_version("0.2.1"):
+        info_dict["execution_summary"]["working_directory"] = ""
+        return
```

Comment:
Most likely there would later be more of `if`s and not necessarily elif's , so might be worth right away to 

```suggestion
```